packages/knip/fixtures/plugins/tsx/node_modules/tsx/package.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "tsx",
+    "bin": ""
```

Comment:
What I typically do is add an empty `index.js` file with `"bin": "./index.js"`. 

Knip looks here for the `tsx` executable referenced in `package.json#scripts` and understands the `tsx` dependency is used.

Reply:
fixed